Extensions/people_notifier.js

```diff
@@ -360,6 +367,48 @@ XKit.extensions.people_notifier = new Object({
 			$(this).find('.count').css('marginRight', '0');
 		});
 
+		var dragging;
+		var dragtarget;
+		var half_div_height = 13;
```

Comment:
Since this is only used inside the `drop` handler, I'd move this inside there as well.